rotkehlchen/chain/manager.py

```diff
@@ -1248,3 +1248,14 @@ def get_eth2_history_events(
                 ))
 
         return defi_events
+
+    @protect_with_lock()
+    @cache_response_timewise()
+    def get_loopring_balances(self) -> Dict[ChecksumEthAddress, Dict[Asset, Balance]]:
+        """Query loopring balances if the module is activated"""
+        # Check if the loopring module is activated
+        loopring_module = self.get_module('loopring')
+        if loopring_module:
+            return loopring_module.get_balances(addresses=self.accounts.eth)
```

Comment:
As per the call I linked above you should only query addresses designated to be queried for loopring.

`self.chain_manager.queried_addresses_for_module('loopring')`